inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
I don't understand what you're saying. These functions do come from is_Ligature.c as of this pull request, right? If not, then the need for a comment here is even greater due to the non-obviousness of the definition sources.